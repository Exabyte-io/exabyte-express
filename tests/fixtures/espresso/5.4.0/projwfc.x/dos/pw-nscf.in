version https://git-lfs.github.com/spec/v1
oid sha256:a676dba78abf77e56470ec7ca64b0f1563d7acab44b76f881bc500d46d37122b
size 1103
