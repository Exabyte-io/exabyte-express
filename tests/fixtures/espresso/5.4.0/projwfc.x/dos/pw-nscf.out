version https://git-lfs.github.com/spec/v1
oid sha256:8a95f594cbbd0c8b9973fc9a6c06fd19093a624d3fdf7e446f296d51ade24748
size 8025
